src/python/tests/CMakeLists.txt

```diff
@@ -38,10 +38,21 @@ set(python_tests_robot
   testAmazonPod.py
 )
 
-# these tests use Atlas v5 robot model
+# Tests are run by default on the default robot
+#
+# If intending to test on a particular robot only, please set _extra_args to the
+# director argument for that robot.
+#
+# Tests can also be run on multiple robots by setting an additional
+# _extra_args_{n} to the director argument for the n-th robot to test.
```

Comment:
This is fine.  I was trying to say that extra_args isn't limited to selecting robots.  For example, a perception test that doesn't load a robot might use extra_args to select an input data set to test with.